### PR TITLE
Major updates to readme

### PR DESCRIPTION
**Pull request by matluuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -420,31 +279,215 @@ The temperature data is stored in files under data folder. One file is made for
...
+Set up Fit IoT-Lab experiment with two nodes for device firmware and border router:
 
 1. Connect to grenoble.iot-lab.info IF not already:
 
     ```batch
     ssh <username>@grenoble.iot-lab.info
     ```
 
-2. Change into the miniproject-1 repository working directory:
+2. Submit an experiment for two nodes at grenoble for 20 minutes:
```
Comment (toukokinnunen) [inline README.md:356]: experiment time should be longer for example 60 min

File: README.md [hunk 2]
```diff
@@ -8,12 +8,29 @@
 
 The project uses [Fit IoT-Lab](https://www.iot-lab.info/) remote testbed to book IoT-devices, to program them, interact with them and control them.
 
+Simple python CoAp server is used to receive temperature data from iot-lab node. The server writes the temperature data to a file, where it could be shown.
+
 ### Table of Contents
 
 - [Getting Started](#getting-started)
```
Comment (toukokinnunen) [inline README.md:15]: delete this

File: README.md [hunk 3]
```diff
@@ -22,165 +39,7 @@ The project uses [Fit IoT-Lab](https://www.iot-lab.info/) remote testbed to book
 
 ### Prerequisites
```
Comment (toukokinnunen) [inline README.md:40]: move this under "set up iot lab nodes"

File: README.md [hunk 4]
```diff
@@ -420,31 +279,215 @@ The temperature data is stored in files under data folder. One file is made for
...
+
+To flash the application to iot-lab node, follow these guidelines:
+
+1. Open new ssh connection to grenoble.iot-lab.info:
+
+    ```batch
+    ssh <username>@grenoble.iot-lab.info
+    ```
+
+2. Set source:
```
Comment (toukokinnunen) [inline README.md:470]: change places between 2 and 3

File: README.md [hunk 5]
```diff
@@ -420,31 +279,215 @@ The temperature data is stored in files under data folder. One file is made for
...
     cd IoT-miniproject-1/app
     ```
 
-3. Flash the firmware into the experiment node:
+4. Flash the firmware into the experiment node. Use different `<board-id>` than for border router.
 
     ```bash
-    make IOTLAB_NODE=m3-114 DEFAULT_CHANNEL=18 DEFAULT_PAN_ID=0xbcb4 flash 
+    make IOTLAB_NODE=m3-<board-id> DEFAULT_CHANNEL=18 DEFAULT_PAN_ID=0xbcb4 flash 
     ```
```
Comment (toukokinnunen) [inline README.md:486]: add nc command and new terminal for that

File: README.md (hunk 2 above)
Comment (matluuk) [reply README.md:15]: Im no goinng to delete this.

File: README.md (hunk 1 above)
Comment (matluuk) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (matluuk) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (matluuk) [reply README.md:482]: Added

File: README.md (hunk 4 above)
Comment (matluuk) [reply README.md]: Done

**Pull request merged**